src/api/supabase/queries/user_queries.ts

```diff
@@ -39,7 +39,7 @@ export async function fetchUserByUUID(uuid: string) {
     const { data: user, error } = await supabase
```

Comment:
https://github.com/calblueprint/shanti-project/blob/kevincai/new_back/src/api/supabase/queries/user_queries.ts

look at this to use auth to grab the user (still needa test a little)

Reply:
Hi Kevin, thanks for the comments! Super helpful. I don't think I wrote this function so I might pass it to you but I have resolved the rest of the comments